shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/LedgerState.hs

```diff
@@ -633,65 +634,59 @@ genesisState genDelegs0 utxo0 =
     dState = emptyDState {_genDelegs = GenDelegs genDelegs0}
 
 -- | Implementation of abstract transaction size
-txsize :: Tx crypto -> Integer
+txsize :: Tx crypto v -> Integer
 txsize = fromIntegral . BSL.length . txFullBytes
 
 -- | Convenience Function to bound the txsize function.
 -- | It can be helpful for coin selection.
-txsizeBound :: forall crypto. (Crypto crypto) => Tx crypto -> Integer
+txsizeBound :: forall crypto v. (CV crypto v) => Tx crypto v -> Integer
 txsizeBound tx = numInputs * inputSize + numOutputs * outputSize + rest
```

Comment:
I think we might be able to get rid of this function altogether. which would mean we could also clean up the transaction type (it stores the input and output bytes so that `txsizeBound` can remove them). we can do this in a separate PR.